.gitignore

```diff
@@ -1,2 +1,14 @@
-build*
-code-gallery
+/*build*
+/code-gallery
+/doc/news/changes/changes.h.in
```

Comment:
why do you want to ignore this file? how does it get there?

Reply:
maybe a left-over from old days. Can remove